### ISSUE TITLE
An honest question

### ISSUE THREAD

**Issue by namewithnomeaning** [holo-gfx/mangadex#88]
I know I may be a retard as I am kinda brigading with my new account but I honestly want to know some things like was this account behind the hacking? Have ips or personal information been leaked? Or if not too dumb a question why was there a hacking in the first place? 
You Githubbers already know something we (and by extension I) don't. I may not be using this issue tab properly but please indulge an idiot

**Comment by xerxxlugner** [comment]
Please read all the other issues before opening a new one. I think it is thoroughly explained there by experienced devs. 

TLDR; Dogshit code, extremely bad code management, biased story.. ?  Hacker may or may not be in the wrong?

**Comment by sdarkpaladin** [comment]
TL;DR as someone who likes Mangadex and have been following the story/read through all the comments.

1) Mangadex Site has vulnerabilities

2) Other codes pointed it out to Mangadex (Henceforth MD) Staff

3) MD Staff ignored/didn't fix it.

4) MD got hacked in the past. Said they fixed it. But problem still exist. 

5) holo-gfx messaged them about vulnerabilities. MD Staff claims holo-gfx demanded payment of a lot of money. MD Staff ignored him.

6) holo-gfx went in and did mischief like deleting translation groups. MD Staff took MD down to patch vulnerabilities.

7) Vulnerabilities still exists.

8) holo-gfx tried exploiting vulnerabilities again but was caught by MD Staff who took the entire site down.

9) holo-gfx dumped the entier code of MD in github (here) but none of the database data. (Your personal data should be safe as long as holo-gfx don't publicly release it. But even if he did it will be your email and password combination for MD only. Unless you use the same combination elsewhere, you should only be receiving spam mail at best.)

10) MD Staff final decision is to close the whole site and hire people to help re-code a new site.

11) Most comments here from people who actually code are saying MD's original code is absolute garbage trash and shouldn't have existed. Reaction to whether future code will be better is mixed.

So far, holo-gfx has not said anything about his intentions beyond taunting MD staff and asking for money. MD staff has published an official statement but is only one side of the story.

If there are anything that I did not cover or covered wrongly please inform me and I'll change it.

**Comment by xerxxlugner** [comment]
> 
> So far, holo-gfx has not said anything about his intentions beyond taunting MD staff and asking for money. MD staff has published an official statement but is only one side of the story.
As far as we know, there is no proof of the hacker asking for a sum or money/ransom.

**Comment by sdarkpaladin** [comment]
> > So far, holo-gfx has not said anything about his intentions beyond taunting MD staff and asking for money. MD staff has published an official statement but is only one side of the story.
> 
> As far as we know, there is no proof of the hacker asking for a sum or money/ransom.

Aye, but the hacker has not refuted it either. I'll add allegedly into the list then.
